api/v1alpha1/status.go

```diff
@@ -122,8 +122,8 @@ type RuntimeStatus struct {
 	APIGatewayStatus *APIGatewayStatus `json:"apiGateway,omitempty"`
 
 	// MountTime represents time last mount happened
-	// if Mounttime is early than master starting time, remount will be required
-	MountTime metav1.Time `json:"mountTime,omitempty"`
+	// if Mounttime is earlier than master starting time, remount will be required
+	MountTime *metav1.Time `json:"mountTime,omitempty"`
```

Comment:
I think several other places need to be updated together: 

1. [mountTime](https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ddc/alluxio/ufs_internal.go#L326) should be set as follows, otherwise it may cause `invalid memory address or nil pointer dereference`: 

```
runtimeToUpdate.Status.MountTime = &metav1.Time{Time: time.Now()}
```

2. [mountTime](https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ddc/alluxio/ufs.go#L156) should check nil :

```
if startedAt != nil && runtime.Status.MountTime != nil && runtime.Status.MountTime.Before(startedAt) {
```

Reply:
In addition, CRD should be upgraded as well.

```
make manifests
make update-crd
```